src/core/manager.c

```diff
@@ -2749,6 +2749,10 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                 r = unit_deserialize(u, f, fds);
                 if (r < 0)
                         goto finish;
+
+                r = unit_realize_cgroup(u);
```

Comment:
Some cgroup settings may be changed. Do we really want to apply those changes immediately?

By the way, there is a similar discussion about `Exec*`: https://github.com/systemd/systemd/pull/5354#issuecomment-287316131